hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapLoaderTest.java

```diff
@@ -72,6 +72,7 @@ public void testSenderAndBackupTerminates_AfterInitialLoad() throws InterruptedE
         HazelcastInstance[] ownerAndReplicas = findOwnerAndReplicas(instances, name);
         ownerAndReplicas[0].getLifecycleService().terminate();
         ownerAndReplicas[1].getLifecycleService().terminate();
+        waitUntilActive(ownerAndReplicas[3], 15);
```

Comment:
I don't understand how this helps? When cluster gets out of `ACTIVE` state in this test?